.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Run Jest Tests",
+        "program": "${workspaceFolder}/node_modules/jest/bin/jest.js",
+        "args": [
+          "${workspaceFolder}/test/example.spec.js",
```

Comment:
Awesome setup ! 
I think this for templating, right ?

Reply:
Yes, exactly, it is for run a specific test file when we have many test files